lib/public/components/Table/content.js

```diff
@@ -10,31 +10,73 @@
  * granted to it by virtue of its status as an Intergovernmental Organization
  * or submit itself to any jurisdiction.
  */
-
-import { h } from '/js/src/index.js';
+import { h, iconPlus, iconMinus } from '/js/src/index.js';
 
 /**
  * Renders a single row with content
  * @param {Object} value The key value potentially containing a formatter function
  * @param {String} text The content text
+ * @param {String} rowId The current rowId
+ * @param {Object} model The global model object
  * @return {vnode} A single table cell containing (formatted) text
  */
-const row = (value, text) => {
-    const formatted = value && value.format ? value.format(text) : text;
-    return h('td#', formatted);
+const row = (value, text, rowId, model) => {
+    const formattedText = value && value.format ? value.format(text) : text;
+
+    const columnId = `${rowId}-${value.name.toLowerCase()}`;
+    const canExpand = model.logs.canColumnExpand(rowId, value.name);
+    const isExpanded = model.logs.isColumnExpanded(rowId, value.name);
+    const base = `.${value.size}#${columnId}`;
+
+    return h(`td${base}`, {
+        onupdate: () => {
+            if (value.expand) {
+                const element = model.document.getElementById(`${columnId}-text`);
+                const minimalHeight = model.logs.getMinimalColumnHeight(rowId, value.name);
+                if (
+                    !canExpand && (element.scrollHeight > element.offsetHeight || element.scrollHeight > minimalHeight)
```

Comment:
Store the (sub-)expression in a variable instead of this weird format

Reply:
Done